galter_subjects_utils/writer.py

```diff
@@ -26,3 +29,89 @@ def write_jsonl(
             f.write("\n")
 
     return filepath
+
+
+class SubjectDeltaLogger:
+    """Convenience logger for delta operations applied to records."""
+
+    def __init__(self, filepath=None):
+        """Constructor."""
+        if not filepath:
+            # In memory
+            self.f = StringIO()
```

Comment:
Really a convenience for tests. It has the same interface as File objects which allows us to do nice things.